### PR TITLE
Port legacy aws-amplify-react-native docs

### PR DESCRIPTION
**Pull request by reesscot**
*Description of changes:*
Port legacy `aws-amplify-react-native` docs to `README.md`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: legacy/aws-amplify-react-native/README.md [hunk 1]
```diff
@@ -3,3 +3,952 @@
...
+
+---
+
+# Storage
+
+## S3 Album
+
+`S3Album` renders a list of `S3Image` and `S3Text` objects:
+
+![Image](/images/S3Album_and_code.png)
```
Comment (ErikCH) [inline legacy/aws-amplify-react-native/README.md:494]: Does this exist?

Comment (ericclemmons) [reply legacy/aws-amplify-react-native/README.md:494]: @reesscot had to do the same for #790

Comment (reesscot) [reply legacy/aws-amplify-react-native/README.md:494]: Yes, will update now

Comment (reesscot) [reply legacy/aws-amplify-react-native/README.md]: Fixed:
Screen Shot 2021-11-18 at 5 19 28 PM

**Review by ErikCH — APPROVED** [review]

**Pull request merged**